icekit/contrib/navigation/abstract_models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from django.utils.six import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+from fluent_contents.models import PlaceholderField, ContentItem
+from icekit.fields import ICEkitURLField
+
+
+@python_2_unicode_compatible
+class AbstractNavigation(models.Model):
+    name = models.CharField(max_length=255)
+    slug = models.SlugField()
+    pre_html = models.TextField(blank=True, default='<nav><ul>')
```

Comment:
hrm, I suppose the actual HTML should be `<nav id='{{ slug }}'>` or some such, and hence rendered as a template. Can we either accept template markup in these fields, or make them blank and hardcode the nav/ul markup in a template.